### ISSUE TITLE
Use a button instead of a checkbox for the switch component

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#149]
### Problem

[atoms/atoms.switch](https://a-ui.github.io/core_branding_scss/#atoms-switch)

Changing the status of a switch should immediately take effect. Something is on or off. You don't have to 'submit' the change. For that reason, the use of an input type checkbox is not appriopriate for a toggle switch. A checkbox is a form input that should be submitted after checking of unchecking the checkbox. It should not be submitted on input. If it does, it is unexpected behaviour and would fail success criterium 3.2.2 On Input. 

```html
<div class="a-input">
  <label class="a-input__label">Switch</label>
  <div class="a-switch">
    <label class="a-switch__label">Off</label>
    <div class="a-switch__toggle">
      <input type="checkbox" name="switch1" id="switch1" role="switch" aria-checked="false">
      <label for="switch1"><span class="u-screen-reader-only">Toggle switch</span></label>
    </div>
    <label class="a-switch__label">On</label>
  </div>
</div>
```

The current code has 4 different label elements. None is programmatically associated with the input.

```html
<label class="a-input__label">Switch</label>
<label class="a-switch__label">Off</label>
<label for="switch1"><span class="u-screen-reader-only">Toggle switch</span></label>
<label class="a-switch__label">On</label>
```

![20201109-13-00-32](https://user-images.githubusercontent.com/870049/98540114-b6f25500-228d-11eb-868a-beac8c72f8ab.jpg)
![20201109-13-01-11](https://user-images.githubusercontent.com/870049/98540122-b9ed4580-228d-11eb-88d8-780ce93eed7b.jpg)

### Solution

* Use a button instead of a checkbox.
* Use only one label and connect it to the button with a for and id attribute.
* See implementation by Erik Kroes which hides the inactive status.

### Links and articles

* [Erik Kroes: Accessible HTML Toggle Button (Switch)](https://www.erikkroes.nl/blog/accessible-html-toggle-button-switch/)
* [Understanding Success Criterion 3.2.2: On Input](https://www.w3.org/WAI/WCAG21/Understanding/on-input.html)

**Comment by TriangleJuice** [comment]
Fixed in #183
